test/parallel/test-cluster-dgram-1.js

```diff
@@ -28,25 +28,37 @@ const assert = require('assert');
 const cluster = require('cluster');
 const dgram = require('dgram');
 
+function getPort(cb) {
+  const s = dgram.createSocket({ type: 'udp4', reuseAddr: true });
+  s.bind(0, () => {
+    const port = s.address().port;
+    s.close();
+    cb(null, port);
```

Comment:
By the time this port is used by the test, there is a possibility that this would be assigned to someother process by the operating system.